src/pages/public_registration/RegistrationLandingPage.tsx

```diff
@@ -19,14 +19,16 @@ import { ActionPanel } from './ActionPanel';
 import { PublicRegistrationContent } from './PublicRegistrationContent';
 
 export const RegistrationLandingPage = () => {
+  const history = useHistory();
   const { t } = useTranslation();
   const dispatch = useDispatch();
   const { identifier } = useParams<IdentifierParams>();
+  const shouldNotRedirect = new URLSearchParams(history.location.search).has('shouldNotRedirect');
   const user = useSelector((store: RootState) => store.user);
 
   const registrationQuery = useQuery({
-    queryKey: ['registration', identifier],
-    queryFn: () => fetchRegistration(identifier),
+    queryKey: ['registration', identifier, shouldNotRedirect],
+    queryFn: () => fetchRegistration(identifier, shouldNotRedirect),
```

Comment:
Hacken for å stoppe redirecten.

Reply:
Hva gjør APIet annerledes når den får `?doNotRedirect=true`? vurder om det skulle vært tatt med i OpenAPI-spec også

Reply:
Lagd en PR for å få open-api spec oppdatert: https://github.com/BIBSYSDEV/nva-publication-api/pull/1527

Reply:
En dag kommer vil til å angre på dette, og må gjøre mye ekstra jobb for å reimplementere ting her-og-der. Da skal jeg gå på Slabberas å kjøpe meg en deilig kanelbolle :D